src/PDecor.cc

```diff
@@ -1641,6 +1641,20 @@ PDecor::checkSnap(void)
     }
 }
 
+int
+PDecor::getMoveResizeOffset(const Action& action, bool vertical)
+{
+    char *endptr = nullptr;
+    int offset = strtol(action.getParamS(0).c_str(), &endptr, 10);
```

Comment:
Instead of having this parsing here I think it is better to keep it in Config.cc and instead use an extra I param to signal that it is specified in percent.

E.g. action.setParamI(0, 0) for constant and action.setParamI(0, 1) for percent.

Defining some constant or enum ACTION_MOVE_RESIZE_CONSTANT and ACTION_MOVE_RESIZE_PERCENT to make the code more readable.

Reply:
> Instead of having this parsing here I think it is better to keep it in Config.cc and instead use an extra I param to signal that it is specified in percent.
> 
> E.g. action.setParamI(0, 0) for constant and action.setParamI(0, 1) for percent.

Like this?

> Defining some constant or enum ACTION_MOVE_RESIZE_CONSTANT and ACTION_MOVE_RESIZE_PERCENT to make the code more readable.

I'm not sure I understand this part. `ACTION_MOVE_RESIZE` is currently apped to the the "MoveResize" action. Are we introducing new actions ?

Both Config.cc and PDecor.cc are not unit tested yet. I'll have to spend more time to see how to get them in the test suite first before actually adding tests for the new code.